### PR TITLE
Java program to multiply two numbers

### LINKED ISSUE #201: Java Basic Programs
### Kind of Issue

Other

### Description

Adding java programs

- [x] Print An Integer #13 
- [x] Addition of two numbers #248 
- [x] Check Even or Odd Number #267 
- [x] Addition of two binary numbers #367
- [x] Add Two Complex Numbers #368 
- [x] Java Program to Multiply two Numbers #371 
- [x] Java Program to check Leap Year #561 
- [x] Java Program to check whether input character is vowel or consonant #743 
- [x] Java Program to calculate compound interest #570 
- [x] Java Program to calculate simple interest #564 
- [x] Java Program to find quotient and remainder #567 
- [x] Java Program to calculate power of a number #567 
- [ ] ....

### Contributing

Yes

### PR DESCRIPTION
**Pull request by AbhishekJadhav2002**
**Issue:** #201 

### Description

Added Java program to multiply two numbers
fixes #201

### CODE REVIEW

**Comment by AbhishekJadhav2002** [comment]
@uiuxarghya For now I'm providing only one task per request.
Please suggest changes, if any.
As, my request get approved, that is, after getting familiar with the pattern, I will try to complete further tasks as well.

**Comment by uiuxarghya** [comment]
> @uiuxarghya For now I'm providing only one task per request. Please suggest changes, if any. As, my request get approved, that is, after getting familiar with the pattern, I will try to complete further tasks as well.

Well done👍 @AbhishekJadhav2002. 
You have done a great job but it needs certain changes before merging.

File: src/navs/program.js
Comment (uiuxarghya) [inline src/navs/program.js:13]: Add this line to the last in this list, 
You must follow the order of task list #201

File: src/pages/programs/multiply-two-numbers.mdx
Comment (uiuxarghya) [inline src/pages/programs/multiply-two-numbers.mdx:7]: Replace it with
```jsx
import { TipInfo } from '@/components/Tip'
```

File: src/pages/programs/multiply-two-numbers.mdx
Comment (uiuxarghya) [inline src/pages/programs/multiply-two-numbers.mdx:45]: If you know the Scanner class then it would be better to rewrite the code and output as per the Scanner class

File: src/pages/programs/multiply-two-numbers.mdx
Comment (uiuxarghya) [inline src/pages/programs/multiply-two-numbers.mdx:81]: Also, Rewrite this code and output as per the Scanner class.

**Review by uiuxarghya — CHANGES_REQUESTED** [review]

**Comment by AbhishekJadhav2002** [comment]
Also I wanted to ask that, is this project considered for Hacktoberfest @uiuxarghya

**Comment by uiuxarghya** [comment]
> Also I wanted to ask that, is this project considered for Hacktoberfest @uiuxarghya

Yes, it is.

**Review by uiuxarghya — APPROVED** [review]

**Comment by uiuxarghya** [comment]
Thanks a lot @AbhishekJadhav2002  for contributing to this repo. 😃✨

**Pull request merged**
